packages/ui-kit/src/components/rss-feeds.js

```diff
@@ -1,121 +1,78 @@
-      }
-    }
-  }
-`;
-
-const DateWrapper = styled.span`
-  white-space: nowrap;
-`;
+async function fetcher(...args) {
+  const rssParser = new Parser();
+  const promises = args.map(async (feed) => {
+    const feedData = await rssParser.parseURL(feed.url);
+    const refactoredData = feedData.items.map((item) => {
+      return { ...item, feedName: feed.title };
+    });
+    return refactoredData;
+  });
+  return Promise.all(promises).then((data) => {
+    return data;
+  });
+}
 
 const RssFeeds = (props) => {
-  if (!props.url) {
-    const message = 'Must pass prop url to RssFeeds component.';
-    if (process.env.NODE_ENV !== 'production') {
-      return <ContentNotifications.Error>{message}</ContentNotifications.Error>;
-    }
-
+  if (!props.dataSources) {
+    const message = (
+      <ContentNotifications.Error>
+        Must pass a source to RssFeeds component
+      </ContentNotifications.Error>
+    );
     throw new Error(message);
   }
 
-  async function fetcher(...args) {
-    const rssParser = new Parser();
-    const feed = await rssParser.parseURL(args[0]);
-    return feed;
-  }
-
-  // eslint-disable-next-line react-hooks/rules-of-hooks
-  const { data, error } = useSWR(props.url, fetcher);
+  const { data, error } = useSWR(props.dataSources, fetcher);
 
   if (error) {
     console.log(error);
-    return (
+    const message = (
       <ContentNotifications.Error>
-        Error Loading RSS feed from {props.url}
+        Error Loading Data from {props.dataSources}
       </ContentNotifications.Error>
     );
+    throw new Error(message);
   }
 
-  return (
-    <div>
-      {!data ? (
-        <LoadingSpinner size="s">{'Loading feeds'}</LoadingSpinner>
-      ) : (
-        <Table>
-          <thead>
-            <tr>
-              <th colSpan="2">{props.title ? props.title : data.title}</th>
-            </tr>
-          </thead>
-          <tbody>
-            {data.items.map((item, index) => (
-              <tr key={`${item.title}${index}`}>
-                <td
-                  css={css`
-                    width: ${dimensions.widths.rssFeedDateWidth};
-                  `}
-                >
-                  <DateWrapper>
-                    <Link
-                      href={item.link}
-                      css={css`
-                        text-decoration: none;
-                      `}
-                    >
-                      {moment(item.pubDate).format('D MMMM YYYY')}
-                    </Link>
-                  </DateWrapper>
-                </td>
-                <td>{item.title}</td>
-              </tr>
-            ))}
-          </tbody>
-        </Table>
-      )}
-    </div>
-  );
+  if (data) {
+    const oldestEntriesOfFeeds = data.reduce((list, feed) => {
+      return [...list, feed[feed.length - 1]];
+    }, []);
+
+    // we use the newest of the oldest entry of each feed as the last entry in the release note list
+    const lastEntryOfList = oldestEntriesOfFeeds.reduce(
```

Comment:
If you're wondering why we need this logic, take a look at the ticket to see the requirements
https://github.com/commercetools/commercetools-docs-kit/issues/722#issuecomment-718179010